src/TMainConsole.h

```diff
@@ -55,7 +55,7 @@ class TMainConsole : public TConsole
     bool showWindow(const QString& name);
     bool hideWindow(const QString& name);
     bool printWindow(const QString& name, const QString& text);
-    void setProfileName(const QString&);
+    void setProfileName(const QString&) override;
```

Comment:
agree!

Reply:
Well I'd say *that depends*: https://softwareengineering.stackexchange.com/a/52890 In this case where there is only one argument and what it must be is obvious it is cleaner to leave it out - however if there is more than one of a particular type (like say in the prior `printWindow(...)`) then including argument names makes sense.

ℹ️ Not that it is relevant here - but were you aware that not giving an argument name in the definition is another way to avoid the "Unused argument" warning that you'd otherwise have to use the Qt `Q_UNUSED(`...`)` wrapper to silence?

Reply:
I don't agree with this approach. It's less intuitive than naming the arguments.